src/items/use-declarations.md

```diff
@@ -82,13 +82,9 @@ In this example, the module `quux` re-exports two public names defined in
 
 ## `use` Paths
 
-Paths in `use` items must start with a crate name or one of the [path
-qualifiers] `crate`, `self`, `super`, or `::`. `crate` refers to the current
-crate. `self` refers to the current module. `super` refers to the parent
-module. `::` can be used to explicitly refer to a crate, requiring an extern
-crate name to follow.
```

Comment:
The snippet above you're removing seems roughly accurate? Rather than removing it, can we *just add* your note?

Reply:
I think it is misleading, because it seems to imply there is something special about `use` paths or that the first segment is restricted somehow. This hasn't been true since uniform paths (which feels like an eternity ago, but was only in January!).

The `use` path is really any "vaild" path for some definition of "valid".

The "paths" chapter describes what all these qualifiers mean, so I don't think they need to be repeated. 

From my incomplete notes (which are likely outdated), paths can also start with:
- Item? in the current scope.  (Item may be the wrong word.)
- A macro in the macro_use prelude.
- An item in the prelude.
- Built-in type.
- Built-in attribute (but can't be used for some reason).
- Built-in tool module (but can't be used for some reason).

Plus a bunch of restrictions on what *can't* be used (associated items, built-in macros, type parameters, local variables, `Self` types, `impl` items, etc).